src/fhirtypes/ElementDefinition.ts

```diff
@@ -1889,6 +1921,52 @@ export class ElementDefinition {
     return [];
   }
 
+  /**
+   * Checks a StructureDefinition's differential to determine if the profile element extension has been used on
+   * an element.
+   * @param {Object} profileJson - The json reprsentation of this ElementDefinition's structDef
+   * @returns {boolean | string} If the profile element extension is not found, false if returned. If it is found,
+   * the id of the BackBone Element being profiled is returned.
+   */
+  private hasProfileElementExtension(profileJson?: any): boolean | string {
```

Comment:
I think it makes sense to clarify the name of this function and the return type. If we want it to return the `id` of the `BackboneElement` being profiled, let's just make it do that, return only a string, and rename it something like `getProfileElementExtensionTarget`. Or along those lines.

Reply:
I chose to just simplify the return type to only be a boolean, returning `true` if the extension is found and `false` otherwise. Since we're only including support for the profile-element extension on `contentReference` elements, there shouldn't be a need to return the `id` of the `BackboneElement` anyways (it will always be the same as the `contentReferenceId`). When we expand support to include other `BackboneElement`s, it'd probably make sense to just make `getProfileElementExtensionTarget` a separate function.